src/psyclone/domain/common/transformations/invokecall_trans.py

```diff
@@ -246,10 +246,6 @@ def apply(self, call, index, options=None):
                 self._specialise_symbol(type_symbol)
                 calls.append(KernelFunctor.create(type_symbol, args))
 
```

Comment:
pylint complains about unused imports and the access of `_fp2_nodes` at line 234. I think you can just disable the warning for the latter.

Reply:
Please update the copyright date on this file too.

Reply:
Done, done and done. I didn't disable the remaining pylint "arguments-differ" warning but can do if you think that should also be added.